ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -125,78 +129,92 @@ const ServiceDetailsScreen = (props: Props) => {
 
   return (
     <BaseScreenComponent
-      goBack={() => props.navigation.goBack()}
+      goBack={props.navigation.goBack}
```

Comment:
This is causing a bug in the screen: to reproduce, enter "Carta Giovani Nazionale" and try to navigate back.

```suggestion
      goBack={() => props.navigation.goBack()}
```